.github/workflows/linear.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Find or create a Linear Issue
         uses: risc0/action-find-or-create-linear-issue@risc0
         with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
The documentation on https://github.com/ctriolo/action-find-or-create-linear-issue clearly uses the input name `github-token`. I understand that the corresponding secret and env is typically called `GITHUB_TOKEN`, but I'd still follow the actions documentation (and consistency with the `linear-`) here and not change this.